src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -10,6 +10,24 @@ by Ben Inglis:
 
 ### Scanner Hardware
 
+{{ MACROS___make_metadata_table(
+   {
+      "Manufacturer": "RECOMMENDED",
+      "ManufacturersModelName": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+      "StationName": "RECOMMENDED",
+      "SoftwareVersions": "RECOMMENDED",
+      "HardcopyDeviceSoftwareVersion": "DEPRECATED",
+      "MagneticFieldStrength": "RECOMMENDED, but REQUIRED for Arterial Spin Labeling",
+      "ReceiveCoilName": "RECOMMENDED",
+      "ReceiveCoilActiveElements": "RECOMMENDED",
```

Comment:
I think this is how you add on to the description?

```suggestion
      "ReceiveCoilActiveElements": {"RECOMMENDED", "See an example below the table."}
```

Reply:
The code checks for a tuple instead of a set, but other than that it's right.